src/test/java/io/kestra/plugin/ollama/OllamaCLITest.java

```diff
@@ -0,0 +1,47 @@
+package io.kestra.plugin.ollama;
+
+import io.kestra.core.junit.annotations.KestraTest;
+import io.kestra.core.models.property.Property;
+import io.kestra.core.models.tasks.runners.TaskRunnerDetailResult;
+import io.kestra.core.runners.RunContext;
+import io.kestra.core.runners.RunContextFactory;
+import io.kestra.core.utils.TestsUtils;
+import io.kestra.plugin.ollama.cli.OllamaCLI;
+import io.kestra.plugin.scripts.exec.scripts.models.DockerOptions;
+import io.kestra.plugin.scripts.runner.docker.Docker;
+import jakarta.inject.Inject;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+import java.util.Map;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.notNullValue;
+
+
+@KestraTest
+class OllamaCLITest {
+    @Inject
+    private RunContextFactory runContextFactory;
+
+    @Test
+    void shouldExecuteCommandsAndReturnSuccessExitCode() throws Exception {
+        OllamaCLI task = OllamaCLI.builder()
+            .id(OllamaCLI.class.getSimpleName())
+            .type(OllamaCLI.class.getName())
+            .commands(Property.of(List.of(
+                "ollama pull smollm:360m",
+                "ollama run smollm:360m \"What is your name?\" > output.txt"
+            )))
+            .containerImage(Property.of("alpine/ollama"))
```

Comment:
I guess the alpine image is lighter but we should use `ollama/ollama` just like in the actual task to stay close to the production code.